Mean rh

Brian-Jamison opened the pull request:
These are additions to produce the mean 850mb-500mb RH plot.  A new function, field_mean, was introduced in grib.py to compute the mean of any number of fields of the same type, defined in default_specs.py.  Here is an example with its NCL counterpart:

![image](https://user-images.githubusercontent.com/58485074/99713764-efe6b280-2a61-11eb-8ccb-b772f67fd3e3.png)
![image](https://user-images.githubusercontent.com/58485074/99713845-0c82ea80-2a62-11eb-826d-f870c67bbd57.png)

christinaholtNOAA reviewed (changes requested):
It seems something odd happened with the merge from master here. There are a lot of things that are identical or undoing Craig's previous commits. I don't see a merge from master into this branch? How did you incorporate those changes?

Brian-Jamison commented:
I'm sorry, I must have done this incorrectly.  On my branch, I committed my latest changes, then I switched to master and did a git pull, which brought in Craig's precip type changes.  I switched back to my branch, and looked at differences between my branch and master, and changed my code to match master plus added my changes.  After testing, I did 'git add' for that code and committed and pushed.  How should I correct it?

christinaholtNOAA commented on image_lists/hrrr_subset.yml:
```diff
@@ -83,6 +83,7 @@ hourly:
     rh:
       - 2m
       - 850mb
+      - mean
```
This should also show up in any of the other image lists that are relevant...rap, hrrr, etc.

Brian-Jamison replied:
I'll add it in rap.yml and it should be present for most yml files.

christinaholtNOAA commented on adb_graphics/datahandler/grib.py:
```diff
@@ -155,6 +167,9 @@ def get_level(self, field, level, spec):
         # Requested level
+        if level == 'mean':
+            lev = 0
+            return lev
```
These lines should not be needed. Provide `vertical_index: 0` in the default_specs.yml instead.

Brian-Jamison replied:
I did try that and it does work, but I decided to handle it in grib.py rather than having a vertical_index parameter in the default_specs simply because it looks confusing when just looking at the default_specs.

christinaholtNOAA replied:
I think a quick comment in the default_specs.yml will help with confusion, and keep a bunch of level-type logic out of the code and in the config files.

cshartsough reviewed (approved):
looks good.  It's really just grib.py and default_specs.yml - I assume hrrr.yml and hrrr_subset.yml are not being kept up in the repository?

christinaholtNOAA reviewed (changes requested):
This looks pretty good. Thanks for sorting out the merging issues! 

Please take a look at my inline comments below.

Brian-Jamison commented:
This is an example of the new title:
![image](https://user-images.githubusercontent.com/58485074/99884291-5a2a5f00-2bea-11eb-91e2-3d013323c910.png)

cshartsough commented:
That seemed to do the trick.  Good job.

christinaholtNOAA reviewed (approved):
Looks great, @Brian-Jamison!!
